buildenv/jenkins/openjdk_tests

```diff
@@ -13,14 +13,6 @@ def PLATFORM_MAP = [
         'SPEC' : 'linux_aarch64_cmprssptrs',
```

Comment:
Thanks for your fix! We also need to remove the `_cmprssptrs` from all the `SPEC`, as the default will be mixed now. There are a few of them for different platforms.

Reply:
Thanks for the feedback! I'll take a look when I'm back in the office.

Reply:
```suggestion
        'SPEC' : 'linux_aarch64',
```

I'm guessing that this is the correct change. I'm going to move forward by making this assumption throughout this file. If this is not correct, please let me know the right one.

Reply:
OK. I _think_ I've done what you've asked; if there was xl or mixed and I removed at least one, then I removed `_cmprssptrs` from the remaining SPEC. The remaining `_cmprssptrs` are on RISC and zOS (because they still distinguish), and Solaris (because I didn't remove any xl or mixed definitions).

Reply:
Thanks for the update! For Solaris, riscv and zos, I believe you can also remove the _cmprssptrs. @llxia What do you think?

Reply:
Yes, please remove _cmprssptrs for solaris, riscv and zos. Thanks @jeffalder

Reply:
Awesome, thanks, this is done for all as of 1ab18b3.